app/react-native/src/preview/components/TreeView.js

```diff
@@ -0,0 +1,192 @@
+import React from 'react'
+import types from 'prop-types'
+
+import {
+  TouchableOpacity,
+  View,
+} from 'react-native'
+
+class TreeView extends React.PureComponent {
```

Comment:
looks like this comes from https://github.com/zaguiini/react-native-final-tree-view/.  The license is MIT so we should be ok but it'd be nice to add an attribution at the top of the file. Was there any reason we couldn't use the library directly?

Reply:
As I mentioned above I got issue with haste map. I also want to include it as dependency and I'll try to investigate it little bit later.

Reply:
Got it. If you run out of time or can't figure it out, just add a link to the repo at the top of the file with a description of the issues you ran into.